### ISSUE TITLE
Crash when using "check pronounciation" during reviews

### ISSUE THREAD

**Issue by Anthropos888** [ankidroid/Anki-Android#6516]
###### Reproduction Steps

1. Review a card
2. Menu > check pronounciation

###### Expected Result

Actually I don't know what this new feature is doing, but my expectation was that it works without crashing

###### Actual Result

App crash. Report is being sent.

###### Debug info
AnkiDroid Version = 2.12alpha14

Android Version = 9

ACRA UUID = 1f29f610-893a-403f-83d5-25a26ba97fe0

###### Research
*Enter an [x] character to confirm the points below:*

- [x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ x] I have searched for similar existing issues here and on the user forum
- [x ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by mikehardy** [comment]
@Anthropos888 I think there is a permissions problem there #6504 - can you see if you have permissions assigned to AnkiDroid for microphone, and if you do not then assign them manually in application settings (as a workaround / triage) ?

**Comment by Anthropos888** [comment]
@mikehardy Permissions are all set. That can't be the reason.

**Comment by david-allison-1** [comment]
https://couchdb.ankidroid.org/acralyzer/_design/acralyzer/index.html#/reports-browser/user/1f29f610-893a-403f-83d5-25a26ba97fe0

```
java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.LinearLayout.addView(android.view.View)' on a null object reference
at com.ichi2.anki.Reviewer.toggleMicToolBar(Reviewer.java:11)
at com.ichi2.anki.Reviewer.onOptionsItemSelected(Reviewer.java:24)
at android.app.Activity.onMenuItemSelected(Activity.java:3543)
at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:1)
at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:1)
at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:1)
at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:1)
at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:1)
at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:2)
at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:2)
at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:1)
at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:2)
at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:4)
at androidx.appcompat.view.menu.MenuPopup.onItemClick(MenuPopup.java:6)
at android.widget.AdapterView.performItemClick(AdapterView.java:318)
at android.widget.AbsListView.performItemClick(AbsListView.java:1192)
at android.widget.AbsListView$PerformClick.run(AbsListView.java:3169)
at android.os.Handler.handleCallback(Handler.java:874)
at android.os.Handler.dispatchMessage(Handler.java:100)
at android.os.Looper.loop(Looper.java:198)
at android.app.ActivityThread.main(ActivityThread.java:6729)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by Anthropos888** [comment]
Interesting: I just installed the parallel build A (which is very useful btw) and there it works: choosing "check pronounciation" from the menu brings up the permission request, and pressing "allow" shows the voice recording panel.

Then I removed the permissions for the normal Ankidroid version, and chose "check pronounciation" from the menu: it brings up the permission request, and pressing "allow" crashed the app. Strange.

**Comment by Anthropos888** [comment]
Ok, it has to do with full screen.
I switched to full screen (hide system bar and app buttons) in parallel build A and now it also crashes there when choosing "check pronounciation" from the menu

**Comment by mikehardy** [comment]
Nice triage! Thanks @Anthropos888 
Glad you like the parallel alphas too, I had a hunch they would be useful but nice to hear a +1 report

**Comment by david-allison-1** [comment]
Fixed in #6504
